apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-content-palette/dot-content-palette.component.scss

```
@@ -0,0 +1,79 @@
+@use "libs/dot-primeng-theme-styles/src/scss/variables" as *;
+
+:host {
+    width: $content-palette-width;
+    display: flex;
+    flex-direction: column;
+    padding: $spacing-1 0;
+    border-left: 1px solid $gray-lighter;
+    padding: 0 $spacing-1 ;
+}
+
+.dot-content-palette__search {
+    color: $gray;
+    position: sticky;
+    align-self: flex-start;
+    top: 0;
+    width: 100%;
+
+    input {
+        width: 100%;
+        border: none;
+        outline: 0px;
+        background: transparent;
+        padding: $spacing-2 $spacing-5 $spacing-2 $spacing-1;
+        color: $gray;
+    }
+
+    dot-icon {
+        position: absolute;
+        top: 50%;
+        transform: translateY(-50%);
+        right: $spacing-1;
+    }
+}
+
+.dot-content-palette__items {
+
+    overflow: hidden auto;
+
+    div {
+        width: 7.2rem;
+        height: 5.71rem;
+        border: 1px solid $gray-lighter;
+        display: inline-flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        cursor: grab;
+        color: $black;
+        margin-bottom: $spacing-1;
+
+        &:nth-child(even) {
+            margin-left: $spacing-1;
+        }
+
+        &:hover {
+            background: $gray-bg;
+        }
+
+        span {
+            padding: 0 $spacing-3 / 3;
```

```suggestion
            padding: 0 0.25rem;
```

And mode this to the `div`